### PR TITLE
Implemented tests for Banner component

### PR DESCRIPTION
**Pull request by EmanueleMinotto**
Unit tests covering the Banner component.

This is a split of #991

### CODE REVIEW

File: src/Elcodi/Component/Banner/Tests/UnitTest/Services/BannerManagerTest.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * @author Elcodi Team <tech@elcodi.com>
+ */
+
+namespace Elcodi\Component\Banner\Tests\UnitTest\Services;
+
+use Elcodi\Component\Banner\Repository\BannerRepository;
+use Elcodi\Component\Banner\Services\BannerManager;
+use PHPUnit_Framework_TestCase;
+
+class BannerManagerTest extends PHPUnit_Framework_TestCase
```
Comment (mmoreram) [inline src/Elcodi/Component/Banner/Tests/UnitTest/Services/BannerManagerTest.php]: I don't understand the reason of this test

Comment (EmanueleMinotto) [inline src/Elcodi/Component/Banner/Tests/UnitTest/Services/BannerManagerTest.php]: It's a classic unit test, just to test that `BannerManager` continue to have the same specification.
For example: `getBannersFromBannerZoneCode` shouldn't throw an exception, if it does (with a random string) than there's a bug.

File: src/Elcodi/Component/Banner/Tests/UnitTest/Entity/BannerZoneTest.php [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        ];
+    }
+
+    /**
+     * Test the string casting.
+     */
+    public function testToString()
+    {
+        $object = new BannerZone();
+        $name = sha1(rand());
```
Comment (mmoreram) [inline src/Elcodi/Component/Banner/Tests/UnitTest/Entity/BannerZoneTest.php]: why not "name"? both rand and sha1 use resources

File: src/Elcodi/Component/Banner/Tests/UnitTest/Services/BannerManagerTest.php (hunk 1 above)
Comment (mmoreram) [inline src/Elcodi/Component/Banner/Tests/UnitTest/Services/BannerManagerTest.php]: Well, in that case we could over this cases with functional tests

**Comment by EmanueleMinotto** [comment]
@mmoreram removed the `BannerManagerTest` and updated the PR

**Comment by mmoreram** [comment]
Awesome :)

**Pull request merged**